src/styles/tailwind.css

```diff
@@ -3,7 +3,26 @@
 @tailwind utilities;
 
 @layer base {
+  html {
+    @apply text-dark-charcoal m-0 font-normal leading-loose;
+    -moz-osx-font-smoothing: grayscale;
+    font-weight: 400;
+    margin: 0;
+    font-display: swap;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```

Comment:
this is odd, but it _does_ make text look much better in Firefox on OSX. This is usually the default rendering Firefox uses, but I think it gets confused on the Openverse yellow.

Reply:
This is cool, how did you identify this? 

Side note: Tailwind has [font-smoothing utilities](https://tailwindcss.com/docs/font-smoothing#applying-font-smoothing) but they wouldn't allow values to be browser-specific.

Reply:
> This is cool, how did you identify this?

It was already set, and I'm honestly not sure when or why 😓 but removing it quite noticeably makes the font inter artificially thicker in Firefox.